app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
```

Comment:
In the test `test_sum_of_the_parts_should_be_equal_to_value`, the division should be integer division to ensure the sum of parts equals the integer value. Use `//` instead of `/` for integer division.